src/Form/FastlySettingsForm.php

```diff
@@ -244,6 +256,15 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#open' => TRUE,
     ];
 
+    $key_length = (int) $config->get('cache_tag_hash_length') ?: CacheTagsHashInterface::CACHE_TAG_HASH_LENGTH;
+    $form['purge']['purge_options']['cache_tag_hash_length'] = [
+      '#type' => 'number',
```

Comment:
This will save the validation hook.

```suggestion
      '#type' => 'number',
      '#min' => 4,
      '#max' => 5,
```

Reply:
Modified to suggested.